src/pages/edit_resource/contributors_step/CreatorField.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { FC, useRef, useState } from 'react';
```

Comment:
En annen ting er at tekstfeltet rekker å vises enabled før de kjapt etter blir disabled - får du til å legge inn sjekken før tekst-feltet tegnes opp ?

Eller burde vi vist fram kun ren tekst isteden for disabled tekstbox ?

Reply:
Fisket